### ISSUE TITLE
Crash SDK version >= 3.4.0 for TabLayout and TextInputLayout

### ISSUE THREAD

**Issue by debase** [phrase/phrase-android#59]
##  Context
We are using phrase SDK 3.5.0 in our project using R8 with custom proguard rules.

## Observed behavior
Since the [version 3.4.0](https://github.com/phrase/phrase-android/releases/tag/3.4.0) adding support for TabLayout and TextInputLayout we are experiencing **runtime crashes** on our release builds.

```
at java.lang.reflect.Method.invoke(Native Method)
   at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:942)
  Caused by: android.view.InflateException: Binary XML file line #111 in com.lydia.beta:layout/activity_recipients_picker_pay: Binary XML file line #111 in com.lydia.beta:layout/activity_recipients_picker_pay: Error inflating class com.google.android.material.tabs.TabItem
  Caused by: android.view.InflateException: Binary XML file line #111 in com.lydia.beta:layout/activity_recipients_picker_pay: Error inflating class com.google.android.material.tabs.TabItem
  Caused by: java.lang.NoSuchFieldException: No field text in class Lcom/google/android/material/tabs/TabItem; (declaration of 'com.google.android.material.tabs.TabItem' appears in base.apk!classes23.dex)
   at java.lang.Class.getDeclaredField(Native Method)
   at com.phrase.android.sdk.inject.ViewTransformer.d(SourceFile:35)
   at androidx.appcompat.app.PhraseAppCompatDelegate.i(SourceFile:8)
   at androidx.appcompat.app.PhraseAppCompatDelegate$factory2$1.onCreateView(SourceFile:1)
   at android.view.LayoutInflater.tryCreateView(LayoutInflater.java:1065)
   at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1001)
   at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:965)
```

## Expected behavior
App should not crash when SDK don't find field using reflection on TabIItem

Please can you update logic of reflection to simply ignore and/or warn when class field is not retrieved at Runtime ?
In addition can you also add proguard rules on your documentation for this kind of component support using reflection ?

FYI we simply added this on our proguard rules to fix the issue
```-keep class com.google.android.material.** { *; }```

Thanks

**Comment by theSoenke** [comment]
Hi @debase, thanks for the report. That definitely should not happen and we'll take a look at ensuring that the field is present

**Comment by theSoenke** [comment]
@debase this should now be resolved in version 3.6.0 https://github.com/phrase/phrase-android/releases/tag/3.6.0. It was solved by adding `-keep class com.google.android.material.tabs.TabItem { *; }` to the rules. Thanks again for your report
